quoridor-app/app/screen/game.js

```diff
@@ -2,60 +2,109 @@ import React, { useState, useEffect } from "react";
 import { ActivityIndicator, StyleSheet, Text, View } from "react-native";
 
 import Board from "../component/board.js";
-import { createGame } from "../api/gameApi";
+import { createGame, joinGame, movePawn, getPossiblePawnMoves } from "../api/gameApi";
 
 const styles = StyleSheet.create({
   container: {
-    width: "100%",
-    height: "100%",
-    justifyContent: "center",
-    flexDirection: "column",
-    alignItems: "center"
-  }
+    width: '100%',
+    height: '100%',
+    justifyContent: 'center',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
 });
 
-const renderLoading = () => {
-  return (
-    <View>
-      <ActivityIndicator size="large" color="#0000ff" />
-    </View>
-  );
+const renderLoading = () => (
+  <View>
+    <ActivityIndicator size="large" color="#0000ff" />
+  </View>
+);
+
+const renderGame = (game, setMove, possiblesMoves) => (
+  <View>
+    <Text>Player Turn: {game.pawnTurn}</Text>
+    <Board
+      squares={game.board.squares}
+      possibleMoves={possiblesMoves}
+      pawns={game.pawns}
+      onClick={item => handleClickMove(item, setMove)}
+    />
+  </View>
+);
+
+const handleClickMove = (item, setMove) => {
+  setMove(item);
 };
 
-const renderGame = game => {
-  return (
-    <View>
-      <Text>Player Turn: {game.pawnTurn}</Text>
-      <Board squares={game.board.squares} pawns={game.pawns} />
-    </View>
+const getPossibleMoves = async (gameId, setPossibleMoves) => {
+  const response = await getPossiblePawnMoves(gameId);
+  const content = await response.json();
+  if (!response.ok) {
+    console.err(content);
```

Comment:
I'm not sure `console.err` exist. Did you mean `console.error` ?
There are another .err in the code too